packages/runner-ct/src/app/ReporterContainer.tsx

```diff
@@ -20,14 +20,15 @@ export const ReporterContainer = observer(
   function ReporterContainer (props: ReporterContainerProps) {
     if (!props.state.spec) {
       return (
-        <div className='no-spec'>
+        <div className='no-spec' data-cy="reporter">
```

Comment:
Could we use the text inside instead of using a data-cy ?

Reply:
I prefer using datatest-id because it's more resilient to refactors. We have snapshot tests to detect changes in text content.